docs/data-sources/remote_site_id_mappings.md

```diff
@@ -0,0 +1,66 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "Multi-Site"
+layout: "aci"
+page_title: "ACI: aci_remote_site_id_mappings"
+sidebar_current: "docs-aci-data-source-aci_remote_site_id_mappings"
+description: |-
+  Data source for Remote Site Id Mappings
+---
+
+# aci_remote_site_id_mappings #
+
+Data source for Remote Site Id Mappings
+
+## API Information ##
+
+* Class: [fvRemoteId](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvRemoteId/overview)
+
+* Supported in ACI versions: 3.0(1k) and later.
+
+* Distinguished Name Format: `uni/tn-{name}/ctx-{name}/stAsc/site-{siteId}`
+
+## GUI Information ##
+
+* Location: `Not shown in UI`
+
+## Example Usage ##
+
+```hcl
+
+data "aci_remote_site_id_mappings" "example_associated_site" {
+  parent_dn = aci_associated_site.example.id
+  site_id   = "0"
+}
+
+```
+
+## Schema ##
+
+### Required ###
+
+* `parent_dn` - (string) The distinguished name (DN) of the parent object, possible resources:
+  - [aci_associated_site](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/associated_site) ([fvSiteAssociated](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvSiteAssociated/overview))
+* `site_id` (siteId) - (string) Remote Site ID.
```

Comment:
Should we add some more explanation about what this site id actually is and where / how users should decide the input?